ingest/src/test/scala/hydra/ingest/services/IngestionFlowV2Spec.scala

```diff
@@ -172,4 +186,55 @@ final class IngestionFlowV2Spec extends AnyFlatSpec with Matchers {
     IngestionFlowV2.validateKeyAndValueSchemas(key, None) shouldBe a[Right[Throwable,Unit]]
   }
 
+  it should "accept a logical field type of timestamp-millis having a value 0 before timestamp-millis validation cut-off date" in {
+    val testValPayloadV1: String = s"""{"testTimestamp": 0}"""
+    val testRequest = V2IngestRequest(testKeyPayload, testValPayloadV1.some, ValidationStrategy.Strict.some, useSimpleJsonFormat = false)
+
+    ingest(testRequest, testValSchemaForV1.some, testSubjectV1.some, createdDate = dateStringToInstant("20230530")).flatMap { kafkaClient =>
+      kafkaClient.consumeMessages(testSubjectV1.value, "test-consumer", commitOffsets = false).take(1).compile.toList.map { publishedMessages =>
+        val firstMessage = publishedMessages.head
+        (firstMessage._1.toString, firstMessage._2.get.toString) shouldBe(testKeyPayload, testValPayloadV1)
+      }
+    }.unsafeRunSync()
+  }
+
+  it should "accept a logical field type of timestamp-millis having a negative value -2 before timestamp-millis validation cut-off date" in {
+    val testValPayloadV1: String = s"""{"testTimestamp": -2}"""
+    val testRequest = V2IngestRequest(testKeyPayload, testValPayloadV1.some, ValidationStrategy.Strict.some, useSimpleJsonFormat = false)
+
+    ingest(testRequest, testValSchemaForV1.some, testSubjectV1.some, createdDate = dateStringToInstant("20230530")).flatMap { kafkaClient =>
+      kafkaClient.consumeMessages(testSubjectV1.value, "test-consumer", commitOffsets = false).take(1).compile.toList.map { publishedMessages =>
+        val firstMessage = publishedMessages.head
+        (firstMessage._1.toString, firstMessage._2.get.toString) shouldBe(testKeyPayload, testValPayloadV1)
+      }
+    }.unsafeRunSync()
+  }
+
+  it should "throw an AvroConversionAugmentedException if a logical type(timestamp-millis) field is having a value 0 after timestamp-millis validation " +
+    "cut-off date" in {
+    val testValPayloadV1: String = s"""{"testTimestamp": 0}"""
+    val testRequest = V2IngestRequest(testKeyPayload, testValPayloadV1.some, ValidationStrategy.Strict.some, useSimpleJsonFormat = false)
+    the[AvroConversionAugmentedException] thrownBy ingest(testRequest, createdDate = dateStringToInstant("20230730")).unsafeRunSync()
+  }
+
+  it should "throw an AvroConversionAugmentedException if a logical type(timestamp-millis) field is having a negative value -2 after timestamp-millis " +
+    "validation cut-off date" in {
+    val testValPayloadV1: String = s"""{"testTimestamp": -2}"""
+    val testRequest = V2IngestRequest(testKeyPayload, testValPayloadV1.some, ValidationStrategy.Strict.some, useSimpleJsonFormat = false)
+    the[AvroConversionAugmentedException] thrownBy ingest(testRequest, createdDate = dateStringToInstant("20230730")).unsafeRunSync()
```

Comment:
use date and move to common values because it's the same as in test above

Reply:
done.